PurchasesHybridCommonUI.podspec

```diff
@@ -0,0 +1,35 @@
+Pod::Spec.new do |s|
+  s.name             = "PurchasesHybridCommonUI"
+  s.version          = "8.11.2"
+  s.summary          = "Common files for hybrid SDKs for RevenueCat UI"
+
+  s.description      = <<-DESC
+                       Save yourself the hastle of implementing a subscriptions backend. Use RevenueCat instead https://www.revenuecat.com/
+                       DESC
+
+  s.homepage         = "https://www.revenuecat.com/"
+  s.license          =  { :type => 'MIT' }
+  s.author           = { "RevenueCat, Inc." => "support@revenuecat.com" }
+  s.source           = { :git => "https://github.com/revenuecat/purchases-hybrid-common.git", :tag => s.version.to_s }
+  s.documentation_url = "https://docs.revenuecat.com/"
+
+  s.framework      = 'StoreKit'
+  s.framework      = 'SwiftUI'
+
+  s.dependency 'RevenueCat', '4.32.0'
+  s.dependency 'RevenueCatUI', '4.32.0'
+  s.swift_version = '5.7'
+
+  s.ios.deployment_target = '11.0'
+  s.osx.deployment_target = '10.13'
+  s.tvos.deployment_target = '11.0'
+
+  s.pod_target_xcconfig = { 'DEFINES_MODULE' => 'YES' }
+
+  s.source_files = ['ios/PurchasesHybridCommon/PurchasesHybridCommonUI/**/*.{h,m,swift}']
```

Comment:
I do wonder if it would be better to separate into 2 separate folders inside the `ios` folder, so the separation is clearer... I'm ok with this though

Reply:
Good point, let me clean this

Reply:
Actually `ios/PurchasesHybridCommon` is where the Xcode project is, and since `PurchasesHybridCommonUI` is still part of the same project (just a different target), it makes sense that it's inside.